DNN Platform/Library/Data/DataProvider.cs

```diff
@@ -451,12 +451,6 @@ public virtual string GetUnappliedIterations(string version)
 
 		#region Host Settings Methods
 
-		public virtual void AddHostSetting(string SettingName, string SettingValue, bool SettingIsSecure,
```

Comment:
Brian, 
the stored procedures are not part of the API, nor the DataProvider methods.
Only the controllers and objects, which still use same signatures and functionality.